ui/app/AppLayouts/Chat/panels/communities/CommunityMembersTabPanel.qml

```diff
@@ -70,7 +70,7 @@ Item {
 
                 readonly property bool itsMe: model.pubKey.toLowerCase() === userProfile.pubKey.toLowerCase()
                 readonly property bool isHovered: memberItem.sensor.containsMouse
-                readonly property bool canBeBanned: !memberItem.itsMe && model.memberRole !== Constants.memberRole.owner
+                readonly property bool canBeBanned: !memberItem.itsMe && (model.memberRole !== Constants.memberRole.owner || model.memberRole !== Constants.memberRole.admin)
```

Comment:
This always puzzles me :) Are you sure about the OR there? Imho it should go like:

```suggestion
                readonly property bool canBeBanned: !memberItem.itsMe && model.memberRole !== Constants.memberRole.owner && model.memberRole !== Constants.memberRole.admin
```

Reply:
My bad. Sorry I kept having this `IsOwnerOrAdmin()` thing in my head from work in status-go.
You're right this needs change.